### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman10000**
Descriptions can be found in the respective commits

### CODE REVIEW

File: custom/conf/app.example.ini [hunk 1]
```diff
@@ -2188,10 +2188,15 @@ ROUTER = console
 
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
-;[other]
+;[show]
```
Comment (lunny) [inline custom/conf/app.example.ini:2191]: This is a break change.

**Review by lunny — CHANGES_REQUESTED** [review]
Too much break changes. And the PR's aim is not clear.

**Comment by yardenshoham** [comment]
@Justman10000 please read the [contributor guide](https://github.com/go-gitea/gitea/blob/main/CONTRIBUTING.md).

**Comment by Justman10000** [comment]
> Too much break changes. And the PR's aim is not clear.

What a thing? I tested everything locally! It works...

**Comment by yardenshoham** [comment]
@Justman10000 you need to (at least):
1. Name your PR with a short summary of your change
2. Explain why you made the change in the PR description and show what changed (usually with screenshots)
3. Show you are a responsible contributor by reading the [contributor guide](https://github.com/go-gitea/gitea/blob/main/CONTRIBUTING.md)

**Comment by Justman10000** [comment]
> @Justman10000 you need to (at least):
> 
>     1. Name your PR with a short summary of your change
> 
>     2. Explain why you made the change in the PR description and show what changed (usually with screenshots)
> 
>     3. Show you are a responsible contributor by reading the [contributor guide](https://github.com/go-gitea/gitea/blob/main/CONTRIBUTING.md)

1. What's the fucking problem with the PR names? I'm sure you've noticed that all commits have descriptions, which means that no unique title for the PR is possible!
2. Then I would need about 5-10 screenshots! No thanks... Besides, you put up the code, then you will understand it too
3. ...

**Comment by Justman10000** [comment]
> @Justman10000 you need to (at least):
> 
>     1. Name your PR with a short summary of your change
> 
>     2. Explain why you made the change in the PR description and show what changed (usually with screenshots)
> 
>     3. Show you are a responsible contributor by reading the [contributor guide](https://github.com/go-gitea/gitea/blob/main/CONTRIBUTING.md)

1. What's the fucking problem with the PR names? I'm sure you've noticed that all commits have descriptions, which means that no unique title for the PR is possible!
2. Then I would need about 5-10 screenshots! No thanks... Besides, you put up the code, then you will understand it too
3. We all know that no one reads through this....

**Comment by Justman10000** [comment]
As I said, I have tested everything and it works

**Comment by yardenshoham** [comment]
> I'm sure you've noticed that all commits have descriptions, which means that no unique title for the PR is possible!

If you had read the contributing guidelines you would see that the merge commit is named using the PR title. To have a sensible git tree history, we can't have commits say "Update". If you can't give your PR a unique title it means your changeset changes more than one thing. Therefore you should split it into multiple PRs (small diffs are better).

> Then I would need about 5-10 screenshots! No thanks... Besides, you put up the code, then you will understand it too

As the author of a PR, it's your responsibility to convince the maintainers to merge your PR. Attaching screenshots shows your PR adds value and is a good tool to convince the maintainers to merge.

> We all know that no one reads through this....

That's wrong. These guidelines are important to avoid bad PRs and you should read them if you want your PR merged.

> As I said, I have tested everything and it works

[The build is failing.](https://drone.gitea.io/go-gitea/gitea/62300/1/6)

**Comment by wxiaoguang** [comment]
Sorry but I also agree with other maintainers and contributors' comments.

I think the changes of this PR are too many and some of them are lacking of thinking. At the mean time  the complexity it brings seems much bigger than the benefit  it brings.

TBH, I  can not imagine a real scenario that a site admin tries to set these  values to true/false/false/true/true/false to fine tune the UI. Why do they need such combination? If they  want to customize Gitea, there are already a good approach https://docs.gitea.io/en-us/customizing-gitea/ , they could just edit the templates. They could also empty the `footer.tmpl` to get what they want.

And one more thing, although you swear that "I have tested everything and it works", but if "ShowFooter=false", then where are the rendered JS and CSS resources? And please be polite first .... mentioning `f**king` won't help anything.

**Comment by Justman10000** [comment]
> Sorry but I also agree with other maintainers and contributors' comments.
> 
> I think the changes of this PR are too many and some of them are lacking of thinking. At the mean time the complexity it brings seems much bigger than the benefit it brings.
> 
> TBH, I can not imagine a real scenario that a site admin tries to set these values to true/false/false/true/true/false to fine tune the UI. Why do they need such combination? If they want to customize Gitea, there are already a good approach https://docs.gitea.io/en-us/customizing-gitea/ , they could just edit the templates. They could also empty the `footer.tmpl` to get what they want.
> 
> And one more thing, although you swear that "I have tested everything and it works", but if "ShowFooter=false", then where are the rendered JS and CSS resources? And please be polite first .... mentioning `f**king` won't help anything.

It's about being able to hide things like the copyright...

**Comment by wxiaoguang** [comment]
> It's about being able to hide things like the copyright...

Since there are too many UI elements, I think the customized-template  ( https://docs.gitea.io/en-us/customizing-gitea/ ) works better for this requirement.

Gitea is for everyone, each user would have their preference, but Gitea can not continue adding more and more options, it will a mess in the end.

**Comment by Justman10000** [comment]
> > It's about being able to hide things like the copyright...
> 
> Since there are too many UI elements, I think the customized-template ( https://docs.gitea.io/en-us/customizing-gitea/ ) works better for this requirement.
> 
> Gitea is for everyone, each user would have their preference, but Gitea can not continue adding more and more options, it will a mess in the end.

Is it possible to overwrite the copyright somewhere?

**Comment by eeyrjmr** [comment]
> > > It's about being able to hide things like the copyright...
> > 
> > 
> > Since there are too many UI elements, I think the customized-template ( https://docs.gitea.io/en-us/customizing-gitea/ ) works better for this requirement.
> > Gitea is for everyone, each user would have their preference, but Gitea can not continue adding more and more options, it will a mess in the end.
> 
> Is it possible to overwrite the copyright somewhere?

That is what the templates are for. Look at thos page and you will see it isnt stock

https://codeberg.org/

Likewise please review https://docs.gitea.io/en-us/customizing-gitea/

**Comment by Justman10000** [comment]
> > > > It's about being able to hide things like the copyright...
> > > 
> > > 
> > > Since there are too many UI elements, I think the customized-template ( https://docs.gitea.io/en-us/customizing-gitea/ ) works better for this requirement.
> > > Gitea is for everyone, each user would have their preference, but Gitea can not continue adding more and more options, it will a mess in the end.
> > 
> > 
> > Is it possible to overwrite the copyright somewhere?
> 
> That is what the templates are for. Look at this page and you will see it isnt stock
> 
> https://codeberg.org/
> 
> Likewise please review https://docs.gitea.io/en-us/customizing-gitea/

Hmm, interesting! Well, I wasted all my time again

**Pull request closed without merging**